api/mapping/management/commands/create_concept.py

```diff
@@ -0,0 +1,48 @@
+from django.core.management.base import BaseCommand, CommandError
+from data.models.concept import Concept
+from mapping.models import ScanReportConcept, ScanReportField, ScanReportValue
+from mapping.services_rules import (
+    save_mapping_rules,
+    remove_mapping_rules,
+    find_existing_scan_report_concepts
+)
+from django.http import HttpRequest
+
+
+class Command(BaseCommand):
+    help = 'create rules from existing concepts found for a given table id'
+
+    def handle(self, *args, **options):
+
+        male = 8507
+
+        # queries ScanReportConcept table for rows where concept_id=8507
+        query_id = ScanReportConcept.objects.filter(concept_id=male)
+
```

Comment:
Create an empty list here, and then `.append()` each list that you generate below to that list.